docs/INSTALL_WIN.md

```diff
@@ -0,0 +1,57 @@
+# Installing `gsplat` on Windows
+
+Follow these steps to install `gsplat` on Windows.
+
+## Prerequisites
+
+1. Install Visual Studio Build Tools. If MSVC 143 does not work, you may also need to install MSVC 142 for Visual Studio 2019. And your CUDA environment should be set up properly.
+
+2. Activate your conda environment:
+    ```bash
+    conda activate <your_conda_environment>
+    ```
+    Replace `<your_conda_environment>` with the name of your conda environment. For example:
+    ```bash
+    conda activate nerfstudio
```

Comment:
Just to avoid confusion, it might be better to put `conda activate gsplat` as the example